### PR TITLE
removed scrollbar, vuetify is putting style on the HTML tag...

### PR DESCRIPTION
**Pull request by Adriaanse**
solves #53

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE html>
-<html>
+<html style="overflow: hidden">
```
Comment (CindyvdVries) [inline index.html]: Should be in seperate css part. 
Could be done in App.vue instead; 

html {
  overflow: hidden;
}

Comment (Adriaanse) [reply index.html]: I get your point and I tried but that doesn't seem to work ?

Comment (Adriaanse) [reply index.html]: OK, moved the styling to App.vue as suggested, merging branch

Comment (Adriaanse) [reply index.html]: OK, moved to App.vue as suggested

**Pull request merged**